src/Gui/Dialogs/DlgAddPropertyVarSet.cpp

```diff
@@ -285,11 +287,37 @@ void DlgAddPropertyVarSet::removeSelectionEditor()
     }
 }
 
-void DlgAddPropertyVarSet::addEditor(PropertyEditor::PropertyItem* propertyItem)
+void DlgAddPropertyVarSet::addEnumEditor(PropertyItem* propertyItem)
+{
+    auto* values =
+        static_cast<PropertyStringListItem*>(PropertyStringListItem::create());
```

Comment:
I wonder if it would be possible to fix return type of that method to be correct instead of static casting here. Of course not for this PR - but something that you may want to take a look at.